File: Difficulty: Basic/Find if +ve, -ve or 0/find-if-ve-ve-or-0.c
//{ Driver Code Starts
// Initial Template for C

// } Driver Code Ends

#include <stdio.h>

int main() {
    // Declare an integer variable to store user input
    int number;
    
    // Read an integer from the user
    scanf("%d", &number);
    
    // Check if the number is positive, negative, or zero
    if (number > 0) {
        printf("Positive");  // Print "Positive" if the number is greater than zero
    }
    if (number == 0) {
        printf("Zero");  // Print "Zero" if the number is exactly zero
    }
    if (number < 0) {
        printf("Negative");  // Print "Negative" if the number is less than zero
    }
    
    return 0;
}


//{ Driver Code Starts.

// } Driver Code Ends

File: Difficulty: Basic/Find if +ve, -ve or 0/find-if-ve-ve-or-0.js
//{ Driver Code Starts
//Initial Template for javascript

'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => {
    inputString += inputStdin;
});

process.stdin.on('end', _ => {
    inputString = inputString.trim().split('\n').map(string => {
        return string.trim();
    });
    
    main();    
});

function readLine() {
    return inputString[currentLine++];
}

function main() {
    let t = parseInt(readLine());
    let i = 0;
    for(; i<t; i++) {
        let a = parseInt(readLine());
        let obj = new Solution();
        obj.utility(a);
    
console.log("~");
}
}

// } Driver Code Ends

// User function Template for JavaScript

class Solution {
    /**
     * Function to check if a number is positive, negative, or zero.
     * @param {number} number - The input number.
     */
    utility(number) {
        // Check if the number is zero
        if (number === 0) {
            console.log("Zero");
        } 
        // Check if the number is positive
        else if (number > 0) {
            console.log("Positive");
        } 
        // If neither zero nor positive, it must be negative
        else {
            console.log("Negative");
        }
    }
}
